src/meta/src/stream/stream_graph/fragment.rs

```diff
@@ -273,6 +274,7 @@ impl StreamFragmentGraph {
         let fragments: HashMap<_, _> = proto
             .fragments
             .into_iter()
+            .sorted_by(|a, b| a.0.cmp(&b.0))
```

Comment:
May I ask why do we need this?

Reply:
Because iiuc iter visits pairs in arbitrary order. By this sort, we can fill internal table ids in a controlled way, only after which we can build a one-to-one correspondence between the internal table ids.

Reply:
Hmmm. It looks like `table_id_gen` does not rely on the order it's called. All it does is add an offset to the local ID generated by the **frontend**. 🤔

Reply:
Yeah, you are right. I misunderstand it.